docs/organizational-structure.rst

```diff
@@ -45,7 +45,7 @@ Role Contributors
 -----------------
 
 In context of Ansible roles, each role not directly created by DebOps
-Developers needs a Contributor. The role Contributor signs the role using
+Developers is a Contributor. The role Contributor signs the role using
```

Comment:
Now I understand what you mean with this. I would suggest to make this simpler. Each role has a maintainer so the maintainer is the "Role Contributors". I would try to keep the meaning for contributors reserved so that others understand it more easily?